src/helpers/findBreakpoint.js

```diff
@@ -6,9 +6,7 @@
  * @param {number} value
  */
 export function findBreakpoint(stops, value) {
-  return stops
-    .concat([])
-    .sort((a, b) => a - b)
-    .filter(stop => stop >= value)
-    .concat(stop)[0];
+  const filteredBreakpoints = stops.sort((a, b) => a - b)
+                            .filter(stop => stop >= value)
+  return (filteredBreakpoints.length ? filteredBreakpoints : stops)[0]
```

Comment:
can `stops` be something else than an array?

Reply:
in JavaScript, anything :P. In general, the only place it was called is in `getResizeTransformation`, and the input is always an Array. But we should put a check here anyway.